packages/smooth_app/lib/data_models/user_preferences.dart

```diff
@@ -46,6 +46,14 @@ class UserPreferences extends ChangeNotifier {
   /// Attribute group that is not collapsed
   static const String _TAG_ACTIVE_ATTRIBUTE_GROUP = 'activeAttributeGroup';
 
+  /// Panel expanded setting
+  static const String EXPAND_PANEL_NUTRITION_TABLE_ID = 'nutrition_facts_table';
+  static const String EXPAND_PANEL_INGREDIENTS_ID = 'ingredients';
```

Comment:
Those `const` don't belong in the preferences.
They should be private `const` in `KnowledgePanelCard` instead.

Reply:
Thanks, I moved them to KnowledgePanelCard, since I need them in user_perferences, so they remain pubic static const